aten/src/ATen/cuda/CUDABlas.h

```diff
@@ -0,0 +1,248 @@
+#pragma once
```

Comment:
from https://github.com/pytorch/pytorch/pull/20983/files#diff-bf2bf1d6148eb27d3e3e29031655b1deR1

thanks @pearu

Reply:
OK, I guess if we merge this first, @pearu will handle the merge conflicts in his diff